test/unified_format.py

```diff
@@ -124,8 +124,16 @@ def is_run_on_requirement_satisfied(requirement):
             elif client_context.server_parameters[param] != val:
                 params_satisfied = False
 
+    auth = requirement.get('auth')
```

Comment:
I already made this change. Please revert and rebase.

Reply:
👍🏽